src/components/ScreenWrapper.tsx

```diff
@@ -176,7 +177,9 @@ function ScreenWrapper(
     const {isBlurred, setIsBlurred} = useInputBlurContext();
 
     UNSTABLE_usePreventRemove(shouldReturnToOldDot, () => {
-        NativeModules.HybridAppModule?.closeReactNativeApp(false, false);
+        if (HybridAppModule.isHybridApp()) {
+            HybridAppModule.closeReactNativeApp(false, false);
+        }
         setRootStatusBarEnabled(false);
```

Comment:
I think that `setRootStatusBarEnabled(false)` is used only in HybridApp, so we need to move it inside the if statement.

Reply:
That's true but it was already there 🤔 in which scenario this code is invoked?

Reply:
When we go back from travel page to OldDot